app/lending/(1-concepts)/FAQ/page.mdx

```diff
@@ -0,0 +1,185 @@
+Risk boundaries remain isolated at the loan-position level.
+
+
+## Governance & Parameter Changes
+
+### Who controls lending parameters like LTV, caps, and fees?
+Protocol governance.
+Changes follow the same upgrade path as other Chainflip modules (runtime upgrade + validator consensus).
+
+### Who defines new modules such as insurance funds or backstops?
+Governance: responsible for design, capitalisation, safety constraints, and fee routing.
+
+### Can governance pause lending during an emergency?
+Yes. Via Safe Mode, governance can temporarily pause:
+
+- new loan creation
+- liquidations
+- vault egresses
+
+Repayments or top-ups may remain enabled depending on Safe Mode level.
+
+
+## Liquidity & Execution Behaviour
+
+### Are liquidations prioritised over normal swaps?
+
+Not artificially.
+
+All swaps, liquidation or regular, are bundled together before execution. LPs do not target individual swaps, instead they place limit orders at specific blocks, and the protocol consumes those orders as long as price limits (LPP) are respected.
+
+However, liquidation swaps may have a higher chance of being executed because:
+
+- they always use LPP-based pricing
+- they follow market price even if it drops significantly
+- unlike regular swaps, they do not have an absolute minimum price that could cause a refund
+
+### Do liquidations affect normal user swaps?
+No. The JIT AMM ensures:
+
+- stable quoting
+- consistent spreads
+- normal refund behaviour
+- non-blocking execution
+
+### Can liquidation events cause swap failures or delays?
+Not under normal conditions.
+Liquidation and non-liquidation flow coexist without queueing or blocking.
+
+
+## Collateral & Position Management
+
+### Does collateral automatically earn yield?
+No. Collateral is **not** supplied to pools in this version.
+Borrowers pay interest; suppliers earn interest.
+
+### Does auto-top-up guarantee no liquidation?
+It prevents liquidation as long as the account has enough free balance.
+If free balance is insufficient, liquidation proceeds normally.
+
+### Can I borrow against multiple collateral assets?
+Yes. LTV is calculated per-account using the aggregated oracle value of all collateral assets.
+
+
+## Partial Repayments, Partial Liquidations & Edge Cases
+
+### Can I repay partially?
+Yes. Partial repayments reduce outstanding debt and improve LTV.
+
+### Are liquidations partial?
+Yes. Liquidations occur in tranches, not a single large sale.
+This reduces slippage and borrower loss.
+
+### What happens if liquidation recovers more than the debt?
+Excess collateral is returned to the borrower automatically.
+
+### What happens if collateral is insufficient?
+The shortfall is socialised to lenders of **that asset only**.
+
+
+## Integrators & Frontend Behaviour
+
+### Can integrators set custom borrow limits or frontend LTV minimums?
+Yes. Frontends can enforce stricter UX/safety limits.
+
```

Comment:
Not sure if this is necessary to be added here, it also might be missleading too. They can use stricter minimums in the UI yes, but liquidations and the most important actions like topup too will only start executing once the statechain configured parameters are hit so it doesn't make much sense to do this. Lets maybe remove this whole point.